CMP1903_A1_2324/Testing.cs

```diff
@@ -1,20 +1,36 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
-    internal class Testing
+    public class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
+        public static void Test()
+        {
+            Game game = new Game();
+            int[] rolls = new int[3];    // This is an array to store the rolls that happen
 
-        //Method
+            // This below will roll all the dice and store the rolls
+            for (int i = 0; i < 3; i++)
+            {
+                rolls[i] = game.dice[i].Roll();
+            }
+
+            // Testing to ensure that the rolls are between 1 and 6
+            foreach (int roll in rolls)
+            {
+                Debug.Assert(1 <= roll && roll <= 6, "The die roll was out of range: " + roll);
+            }
+
+            // Testing that the sum of every roll is less than a certain amount
+            int total = rolls.Sum();
+            Debug.Assert(3 <= total && total <= 18, "The total value was out of range: " + total);
+
```

Comment:
+ Checks for the sum of all the dice rolls, and compares them against the lowest number and highest number possible (+)